src/platform/intel/cavs/platform.c

```diff
@@ -116,14 +116,11 @@ int platform_init(struct sof *sof)
 {
 	int ret;
 
-	trace_point(TRACE_BOOT_PLATFORM_CLOCK);
-	platform_clock_init(sof);
-
 	trace_point(TRACE_BOOT_PLATFORM_SCHED);
 	scheduler_init_edf();
 
 	/* init low latency timer domain and scheduler */
-	sof->platform_timer_domain = timer_domain_init(sof->platform_timer, PLATFORM_DEFAULT_CLOCK);
+	sof->platform_timer_domain = timer_domain_init(sof->platform_timer, 0);
```

Comment:
The "clk" parameter has no use on Zephyer, so might as well pass a 0.

Reply:
deserves a comment?

Reply:
Done in today's PR revision.